cocotb/share/makefiles/simulators/Makefile.cvc

```diff
@@ -47,48 +47,39 @@ endif
 
 ifeq (, $(CMD))
     $(error "Unable to locate command >$(CMD_BIN)<")
-else
-    CVC_BIN_DIR := $(shell dirname $(CMD))
-    export CVC_BIN_DIR
 endif
 
-#only interpreted mode works for the moment
-CVC_ITERP ?= 1
+CVC_INTERP := $(filter $(EXTRA_ARGS),+interp)
 
-ifeq ($(CVC_ITERP),1)
-    CVC_ARGS    += +interp
-endif
+ifneq (, $(CVC_INTERP)) # Interpreted mode
+`
+$(COCOTB_RESULTS_FILE): $(VERILOG_SOURCES) $(CUSTOM_COMPILE_DEPS) $(CUSTOM_SIM_DEPS)
+	-@rm -f $(COCOTB_RESULTS_FILE)
 
-# Compilation phase
-$(SIM_BUILD)/sim.vvp: $(SIM_BUILD) $(VERILOG_SOURCES) $(CUSTOM_COMPILE_DEPS)
-	MODULE=$(MODULE) \
-	TESTCASE=$(TESTCASE) TOPLEVEL=$(TOPLEVEL) TOPLEVEL_LANG=$(TOPLEVEL_LANG) COCOTB_SIM=1 \
-        $(CMD) $(CVC_ARGS) +acc+2 -o $(SIM_BUILD)/sim.vvp +define+COCOTB_SIM=1 +loadvpi=$(LIB_DIR)/libcocotbvpi_modelsim:vlog_startup_routines_bootstrap $(COMPILE_ARGS) $(EXTRA_ARGS) $(VERILOG_SOURCES)
+	MODULE=$(MODULE) TESTCASE=$(TESTCASE) TOPLEVEL=$(TOPLEVEL) TOPLEVEL_LANG=$(TOPLEVEL_LANG) COCOTB_SIM=1 \
+	$(CMD) +loadvpi=$(LIB_DIR)/libcocotbvpi_modelsim:vlog_startup_routines_bootstrap $(COMPILE_ARGS) $(SIM_ARGS) $(EXTRA_ARGS) $(VERILOG_SOURCES) $(PLUSARGS)
+
+	$(call check_for_results_file)
 
-# Execution phase
-ifeq ($(CVC_ITERP),1)
-    $(COCOTB_RESULTS_FILE): $(SIM_BUILD)/sim.vvp
 else
-    $(COCOTB_RESULTS_FILE): $(SIM_BUILD)/sim.vvp $(CUSTOM_SIM_DEPS)
-	MODULE=$(MODULE) \
-        TESTCASE=$(TESTCASE) TOPLEVEL=$(TOPLEVEL) TOPLEVEL_LANG=$(TOPLEVEL_LANG) COCOTB_SIM=1 \
-        $(SIM_BUILD)/sim.vvp $(SIM_ARGS) $(EXTRA_ARGS) $(PLUSARGS)
-endif
+
+# Compilation phase
+$(SIM_BUILD)/cvcsim: $(SIM_BUILD) $(VERILOG_SOURCES) $(CUSTOM_COMPILE_DEPS)
+	$(CMD) +verbose -o $(SIM_BUILD)/cvcsim $(CVC_ARGS) +loadvpi=$(LIB_DIR)/libcocotbvpi_modelsim:vlog_startup_routines_bootstrap $(COMPILE_ARGS) $(EXTRA_ARGS) $(VERILOG_SOURCES)
 
 # Execution phase
-ifeq ($(CVC_ITERP),1)
-    debug:  $(CUSTOM_SIM_DEPS)
```

Comment:
No more debug target?

Reply:
I think if needed one can now easily highjack the makefile and modify. We should keep Makefiles simple. But this is just opinion.

Reply:
Do you think I should change this?

Reply:
Do we have `debug` in the other makefiles? Seems like we should be consistent.

Reply:
It looks only `icarus` and `ventilator` have `debug`. Consider CVC is not popular.

Reply:
That's a good argument.